packages/adapter-knex/README.md

```diff
@@ -5,21 +5,12 @@ title: Database Adapter - Knex
 
 # Knex Database Adapter
 
-** WARNING: This adapter is under active development and is not production ready. It _will_ drop your entire database every time you connect! **
-
 The [Knex](https://knexjs.org/#changelog) adapter is a general purpose adapter which can be used to connect to a range of different database backends.
 At present, the only fully tested backend is `Postgres`, however Knex gives the potential for `MSSQL`, `MySQL`, `MariaDB`, `SQLite3`, `Oracle`, and `Amazon Redshift` to be supported.
 
 ## Setting Up Your Database
 
-Before running Keystone, you must set up a database, a schema, and a user.
-Assuming you're on MacOS and have Postgres installed the `build-test-db.sh` does this for you:
-
-```sh
-./packages/adapter-knex/build-test-db.sh
-```
-
-Otherwise, you can run the steps manually:
+Before running Keystone, you must set up a database, a schema, and a user. You can run the following commands:
```

Comment:
Can we keep a reference to the `build-test-db.sh` script and include a note that it's only for development within the mono-repo itself?

Reply:
That's not an awesome developer experience. We could link to the script in the repo for people to download and run?